docs/BuildMachineInfo.md

```diff
@@ -0,0 +1,19 @@
+# Build Machine Information
+
+## Windows AMD GPU
+
+* CPU: AMD Ryzen 7 9700X
+* GPU: AMD Radeon RX 9070
+* RAM: 32 GiB
+
+## Windows Intel GPU
+
+* CPU: Intel Xeon <I'll figure out the model>
+* GPU: Intel UHD Graphics 630
+* RAM: <I should figure this out too>
```

Comment:
Waiting on @llvm-beanz to provide the specs for the intel machine.

Reply:
I'm having trouble remoting into the machine. Can you just leave blank lines? I'll hunt it down and update after this PR merges.